spec/integration/rails_grape_spec.rb

```diff
@@ -11,7 +11,8 @@
 if enabled
   require 'action_controller/railtie'
 
-  RSpec.describe 'Rails and Grape integration', :mock_intake, :allow_running_agent do
+  RSpec.describe 'Rails and Grape integration', :mock_intake,
+    :allow_running_agent do
```

Comment:
can we line this up under `:mock_intake`?

Reply:
We can break both? I don't like special situation indenting levels, really.